Dockerfile

```diff
@@ -9,18 +9,15 @@ RUN apt-get -yqq update
 
 WORKDIR /root/bot
 
-ADD https://raw.githubusercontent.com/USDevOps/jenkins_slack_bot/tag_v1/python_mysql.py \
- https://raw.githubusercontent.com/USDevOps/jenkins_slack_bot/tag_v1/slack_cmd_process.py  \
- https://raw.githubusercontent.com/USDevOps/jenkins_slack_bot/tag_v1/slack_message.py  \
- https://raw.githubusercontent.com/USDevOps/jenkins_slack_bot/tag_v1/slackbot.py  \
- https://raw.githubusercontent.com/USDevOps/jenkins_slack_bot/tag_v1/start_app.py /root/bot/
+COPY python_mysql.py slack_cmd_process.py slack_message.py slackbot.py start_app.py /root/bot/
 
-ADD https://raw.githubusercontent.com/USDevOps/jenkins_slack_bot/tag_v1/init.sql /docker-entrypoint-initdb.d/
+COPY init.sql /docker-entrypoint-initdb.d/
 
 RUN chmod 775 /docker-entrypoint-initdb.d/init.sql && chmod -R 775 /root
 
 RUN rm -rf /var/lib/apt/lists/* && \
 rm -rf /var/cache/apk/*
 
 ENV SLACK_BOT_TOKEN="BOT_TOKEN" CHATBOT_NAME="BOT_NAME" \
-APPROVER_SLACK_NAME="APPROVER_SLACK_ID"
+APPROVER_SLACK_NAME="APPROVER_SLACK_ID" JENKINS_URL="JENKINS_URL" \
```

Comment:
In the entrypoint script, verify the variables are set. Otherwise quit with proper error message.

Reply:
I will do that in entry point script slack bot.py and start_app.py which will check if variable is not defined ,then through the error message.